pdns/calidns.cc

```diff
@@ -71,11 +71,13 @@ static void* recvThread(const vector<Socket*>* sockets)
 #if HAVE_RECVMMSG
   vector<struct mmsghdr> buf(100);
   for(auto& m : buf) {
-    fillMSGHdr(&m.msg_hdr, new struct iovec, new char[512], 512, new char[1500], 1500, new ComboAddress("127.0.0.1"));
```

Comment:
I wonder if we used to allocate 512 bytes instead of the usual 256 ones for a reason?

Reply:
No idea. 256 is already generous.